broker/src/main/java/org/apache/rocketmq/broker/processor/SendMessageProcessor.java

```diff
@@ -351,6 +351,21 @@ private RemotingCommand handlePutMessageResult(PutMessageResult putMessageResult
                 response.setRemark(String.format("the message is illegal, maybe msg body or properties length not matched. msg body length limit %dB, msg properties length limit 32KB.",
                     this.brokerController.getMessageStoreConfig().getMaxMessageSize()));
                 break;
+            case WHEEL_TIMER_MSG_ILLEGAL:
+                response.setCode(ResponseCode.WHEEL_TIMER_MSG_ILLEGAL);
+                response.setRemark(String.format("the accurate delay message is illegal, delay time max limit %dms, or check props if setting TIMER_DEL_UNIQKEY or not",
```

Comment:
Maybe "timer message illegal, the delay time should not be bigger than the max delay; or if set del msg, the delay time should be bigger than the current time" better